frontend/src/metabase-lib/Question.ts

```diff
@@ -1090,12 +1090,13 @@ class Question {
     return question;
   }
 
-  // TODO: Fix incorrect Flow signature
-  parameters(): ParameterObject[] {
+  parameters({ collectionPreview } = {}): ParameterObject[] {
     return getCardUiParameters(
       this.card(),
       this.metadata(),
       this._parameterValues,
+      undefined,
+      collectionPreview,
```

Comment:
Not a big fan of comments like this but this function has become ugly. Let's probably use named parameters?

Reply:
you did similar refactoring in master, but it's not backported to 48 (see the base branch)

Reply:
v48 branch in "support state" atm, so I'll not do this refactoring